app/POROS/movie.rb

```diff
@@ -12,4 +12,31 @@ def get_movies(movie_count)
     end
     movie_results
   end
+
+  def get_specific_movie(id)
+    api_key = ENV['MOVIE_API_KEY']
+    conn = Faraday.new(url: 'https://api.themoviedb.org')
+    response = conn.get("/3/movie/#{id}?api_key=#{api_key}&language=en-US")
+    @json = JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def calculate_time(movie)
+    hours = movie[:runtime] / 60
+    minutes = movie[:runtime] % 60
+    hours > 1 ? "#{hours} hours and #{minutes}" : "#{hours} hour and #{minutes}"
+  end
+
+  def find_cast(id)
+    api_key = ENV['MOVIE_API_KEY']
+    conn = Faraday.new(url: 'https://api.themoviedb.org')
```

Comment:
We can definitely pull this conn out into it's own method once we start to refactor